src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -60,23 +60,27 @@
 
 (def ^:private ALL_USERS "All users")
 
+(defn- make-list [input]
+  ;;input may or maynot be a list, make it a list
+  (flatten (conj '() input)))
```

Comment:
Actually, it might make sense to make this nil-safe:
```clojure
(defn- ensure-sequence [input]
  (cond (sequential? input) input
        (nil? input) []
        :else [input]))